Meziantou.Polyfill.Editor/M;System.Collections.Generic.CollectionExtensions.AddRange``1(System.Collections.Generic.List{``0},System.ReadOnlySpan{``0}).cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.Collections.Generic;
+
+static partial class PolyfillExtensions
```

Comment:
Can you use a _classic_ extension method? This allows projects using C# 13 or lower to still benefit from the polyfill. Polyfills using `extension` are disabled when the project is not using at least C# 14.

Reply:
Done!